FueledUtils/CollectionExtensions.swift

```diff
@@ -14,8 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 public extension Collection {
+	///
+	/// Try to get the item at index `index`. If the index is out of bounds, `nil` is returned.
+	///
+	/// Parameter index: The index of the item to tentatively get.
+	/// Returns: The element as a wrapped optional if the `index` is in the `indices` of the collection, `nil` otherwise
+	///
 	public func getSafely(_ index: Self.Index) -> Self.Iterator.Element? {
```

Comment:
should be `getSafely(at index: Self.Index)…` IMO

Reply:
Yep I agree, let me make that change, I'll deprecate the old one